### PR TITLE
data: rajaditya06

### PR DESCRIPTION
**Pull request by rajaditya06**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5902"></a>

### CODE REVIEW

File: data/rajaditya06.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  "bio": "Blockchain Developer|| open source",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/rajaditya02",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Github",
+      "url": "https://twitter.com/rajaditya06",
```
Comment (kumarsonsoff3) [inline data/rajaditya06.json:13]: You might want to use your GitHub URL here. 

```suggestion
      "url": "https://github.com/rajaditya06",
```

**Comment by rajaditya06** [comment]
Thanks @kumarsonsoff3  for pointing it out.

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**